### ISSUE TITLE
[Feature] command to promote `dev` plugins to semver

### ISSUE THREAD

**Issue by mrcjkb** [nvim-neorocks/rocks.nvim#482]
Sometimes plugins that previously only existed as `dev` plugins are published with semver releases later on (e.g. the tree-sitter parsers).
Currently, users have to manually switch from `dev` to semver and there's no indication that this is possible.

`Rocks update` doesn't promote `dev` plugins to semver.
It would be nice if there were an option to do this, e.g.  `:Rocks[!] promote {rock?}` command (with completions for rocks that can be promoted).
Or, alternately, a prompt in `:Rocks[!] update` (or both).

Suggestions for a better name than `promote` are more than welcome!

**Comment by teto** [comment]
I dont think it warrants its own command. Ideally `dev` would be a conscious choice rather than the current way of dealing with missing rockspecs. 
If one wants to switch to the semver version, he could run  `Rocks install <ALREADY_INSTALLED_PLUGIN_TARGETING_DEV> ` and the command might ask the user to  switch to semver.
